src/chart/generateCategoricalChart.tsx

```diff
@@ -313,7 +309,7 @@ export const getAxisMapByAxes = (
   const isCategorical = isCategoricalAxis(layout, axisType);
 
   // Eliminate duplicated axes
-  const axisMap: AxisMap = axes.reduce((result: AxisMap, child: ReactElement): AxisMap => {
+  return axes.reduce((result: AxisMap, child: ReactElement): AxisMap => {
```

Comment:
nit nit, non-blocking: the return here and below slightly decreases readability (at least for me)

Reply:
I personally dislike unnecessary assignments. lol

Reply:
Eh it's preference. I only call it out cause this is a large block of code as the return